Src/BootCamp.Chapter/Player.cs

```diff
@@ -12,50 +12,35 @@ namespace BootCamp.Chapter
     /// </summary>
     public class Player
     {
-        /// <summary>
-        /// Everyone can carry this much weight at least.
-        /// </summary>
-        private const int baseCarryWeight = 30;
-
-        private string _name;
-        private int _hp;
-
-        /// <summary>
-        /// Each point of strength allows extra 10 kg to carry.
-        /// </summary>
-        private int _strenght;
-
+        
         /// <summary>
         /// Player items. There can be multiple of items with same name.
         /// </summary>
         private Inventory _inventory;
-        /// <summary>
-        /// Needed only for the extra task.
-        /// </summary>
-        private Equipment _equipment;
-
         public Player()
         {
+            _inventory = new Inventory();
         }
 
         /// <summary>
         /// Gets all items from player's inventory
         /// </summary>
         public Item[] GetItems()
```

Comment:
Similar comment; refactor away the arrays to use Lists. Then you wont need the `.ToArray()`